jekyll/_cci2/language-php.md

```diff
@@ -56,7 +56,7 @@ version: 2
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/php:7.1-node-browsers # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
Looks like we dropped browsers. If this isn't a real project to run, then I think that's fine. If it is, then the browser tools orb and the browser variant are needed.

Reply:
> If it is, then the browser tools orb and the browser variant are needed.

Do you mean I to drop the node variant and add the browsers variant with browser tool orbs?
Or did you mean I need to add the browsers image + orb in addition to the node variant?